cmake/Modules/FindTinyXML2.cmake

```diff
@@ -13,7 +13,26 @@ if(TinyXML2_FOUND)
       set(TINYXML2_LIBRARY tinyxml2)
     elseif(TARGET tinyxml2::tinyxml2)
       set(TINYXML2_LIBRARY tinyxml2::tinyxml2)
+    else()
+      message(FATAL_WARNING "Unable to determine target for TinyXML2")
```

Comment:
This shouldn't print a warning unconditionally if the caller passed `QUIET` / `REQUIRED`.

Reply:
should be addressed in https://github.com/ros2/tinyxml2_vendor/pull/8/commits/78e8ae16a784ff462c46c72df8f138bd83608b85